src/assets/Doneth.sol.txt

```diff
@@ -58,8 +55,38 @@ contract Doneth {
         addMember(msg.sender, 1, true, _founderName);
     }
 
+    mapping(address => ERC20) public tokens;
```

Comment:
Should this be mapping(address => Token)?

I see below that you are doing:
"tokens[token].totalWithdrawn = tokens[token].totalWithdrawn.add(amount);"

This leads me to assume you are expecting this mapping to be from an address of an ERC20 token contract to a Token struct that basically just holds the total withdrawn for that token.

Reply:
nice catch, ERC20 struct should be Token struct

Reply:
also you're correct again about what the values in the struct mean